README.md

```diff
@@ -1367,14 +1392,15 @@ The following configuration options can **only** be included in `token.getWithou
 | :-------: | ----------|
 | `sessionToken` | Specify an Okta sessionToken to skip reauthentication when the user already authenticated using the Authentication Flow. |
 | `responseMode` | Specify how the authorization response should be returned. You will generally not need to set this unless you want to override the default values for `token.getWithRedirect`. See [Parameter Details](https://developer.okta.com/docs/api/resources/oidc#parameter-details) for a list of available modes. |
-| `responseType` | Specify the [response type](https://developer.okta.com/docs/api/resources/oidc#request-parameters) for OIDC authentication. Defaults to `id_token`. |
+| `responseType` | Specify the [response type](https://developer.okta.com/docs/api/resources/oidc#request-parameters) for OIDC authentication. Defaults to `id_token`. Supported values are `id_token` and `token`. |
```

Comment:
We also need to support `code`. To kick off the PKCE flow the developer needs to pass in the grant type of `authorization_code` and the response type is **required** to be `code`. The only thing we need to restrict is the combination of response types when using PKCE.

Suggestion:
> Supported values are `id_token`, `token`, and `code`. Hybrid flows (`code token`, `code id_token`) are not supported.